firebase-perf/src/main/java/com/google/firebase/perf/logging/ConsoleUrlGenerator.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.logging;
+
+public final class ConsoleUrlGenerator {
```

Comment:
Any unit tests for this? Lets make sure to create some to test if this is working as intended.

Reply:
Any suggestions on what tests to cover? I am not sure how to test logging functions.

Reply:
We could just have static checks for the URLs that are generated from the URLGenerator. Just some straightforward checks to see the URL is as expected for the provided projectID and AppID would be sufficient.

Reply:
Added some straightforward tests.